app/javascript/pages/assets/data/temp-cal-data.json

```diff
@@ -17,6 +17,15 @@
     "hoverimage": "/files/gallery_7.png",
     "explanation": ""
   },
+  {
+    "year": 2020,
+    "title": "A Region Under Construction",
+    "month": "March",
+    "url": "/calendar/2020/march",
+    "image": "/files/february2020.png",
```

Comment:
Are we planning to update this before final merge?

Reply:
Yup, I've got a list of smaller surface tweaks to pull in tomorrow (gallery image, homepage narrative, and updated visualization visual)